app/main/lib/similarity.py

```diff
@@ -93,6 +104,12 @@ def add_item(item, similarity_type):
   app.logger.info(f"[Alegre Similarity] [Item {item}, Similarity type: {similarity_type}] response for add was {response}")
   return response
 
+def callback_add_item(item, similarity_type):
+  if similarity_type == "audio":
+      response = audio_model().add(item)
+  app.logger.info(f"[Alegre Similarity] CallbackAddItem: [Item {item}, Similarity type: {similarity_type}] Response looks like {response}")
+  return response
```

Comment:
add an `else` and log an error whenever `similarity_type` is anything other than `audio`

Reply:
bump. I think we should still log an error here when `similarity_type` is not `audio`